sensor/tests/helper/helper.go

```diff
@@ -138,29 +140,32 @@ type TestContext struct {
 	archivedMessages [][]*central.MsgFromSensor
 }
 
-// DefaultCentralConfig hold default values when starting local sensor in tests.
-func DefaultCentralConfig() CentralConfig {
+// DefaultConfig hold default values when starting local sensor in tests.
+func DefaultConfig() Config {
```

Comment:
Out of curiosity (and without any prior knowledge/context): Why is it that this was renamed away from `CentralConfig`?

Reply:
At first this structure held only configuration for Central (initial policies and such) but now that there's also Sensor configuration I think the name `CentralConfig` could be confusing.